clusterloader2/testing/load/config.yaml

```diff
@@ -163,6 +164,12 @@ steps:
       action: start
       checkProbesReadyTimeout: {{$PROBE_MEASUREMENTS_CHECK_PROBES_READY_TIMEOUT}}
       replicasPerProbe: {{AddInt 2 (DivideInt .Nodes 100)}}
+  {{if $PROMETHEUS_SCRAPE_KUBE_STATE_METRICS}}
```

Comment:
nitpick: do you want to keep them in the same order in both actions?

kube_proxy comes after here in the start action, but before in the gather action

Reply:
Done, thanks.